po/de.po

```diff
@@ -3964,8 +3967,8 @@ msgid ""
 "The password that needs to be combined with the OTP PIN:\n"
 "        <b>{{ lostResult.password }}</b>"
 msgstr ""
-"Das Passwort, das mit der alten OTP-PIN kombiniert werden muss: "
-"<b>{{ lostResult.password }}</b>"
+"Das Passwort, das mit der alten OTP-PIN kombiniert werden muss: <b>{{ "
+"lostResult.password }}</b>"
```

Comment:
Bad linebreak!?

Reply:
Luckily HTML ignores additional spaces and line-breaks:
![grafik](https://github.com/privacyidea/privacyidea/assets/37443810/6f65538e-ea9c-4301-bad8-f08b0abcfc19)